history/characters/310000_patron_characters.txt

```diff
@@ -835,4 +835,104 @@
 		religion=scarlet_light
 	}
 	648.1.5={ death=yes }
+}
+
+#dynasty=125040
+310400={ #Count #Spartan
+	name=Edward
+	dynasty=125040
+	dna="0djfj0f0ab0"
+	properties="0l00fk0000000000000000"
+	culture=gilnean religion=holy_light
+	martial=5 diplomacy=5 stewardship=5 intrigue=5 learning=5
+	effect={add_patron_trait_character_effect=yes}
+	disallow_random_traits = yes
+	559.12.15={ birth=yes add_trait=creature_human }
+	580.1.1={
+		add_trait=intricate_webweaver
+		add_trait=class_rogue_6
+		add_trait=shrewd
+		add_trait=strategist
+		add_trait=falconer add_trait=ambitious add_trait=proud add_trait=stubborn add_trait=greedy add_trait=envious add_trait=cynical add_trait=patient add_trait=ravager
```

Comment:
This seems correct, but for some reason, Ravager isn't there. I would check requirement for trait:
potential = {
		OR = {
			# Warcraft
			religion_group = vrykul_religion_group
			culture = pirate_culture
			# religion = norse_pagan
			# religion = norse_pagan_reformed
			
			any_owned_bloodline = {
				has_bloodline_flag = bloodline_viking
			}
			AND = {
				has_religion_feature = religion_adventuring
				has_religion_feature = religion_seafaring
			}
		}
	}
If I change his religion in this file to for. ex. Odyn, which belongs to vrykul_religion_group, it works, but only as main religion, secret doesn't affect it.

Reply:
Nah, it was a leftover after the testing... unless we want Gilnean to be sea raiders.
Cleared it in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/commit/7a27f80a0148262ccbbd6942118b3d6fb3799405.